uber/site_sections/signups.py

```diff
@@ -81,6 +81,7 @@ def hotel_requests(self, session, message='', decline=None, **params):
             raise HTTPRedirect('index?message={}', 'Only Staffers can request hotel space')
         requests = session.hotel_requests(params, checkgroups=['nights'], restricted=True)
         if 'attendee_id' in params:
```

Comment:
One change we made this year was adding the ``admin_only`` parameter to the ``Column`` function in ``models.py``, which had the side effect of making it so we don't allow attendees to alter foreign keys anymore.  This actually makes sense; now that I think about it, someone who knew another attendee's id could have theoretically submitted or even edited their hotel requests until now!